src/main/java/hw3/pages/AbstractPage.java

```diff
@@ -0,0 +1,19 @@
+package hw3.pages;
+
+import hw3.components.HeaderMenu;
+import hw3.components.SideBarMenu;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class AbstractPage {
+
+  protected WebDriver driver;
+  protected SideBarMenu sideBarMenu;
+  protected HeaderMenu headerMenu;
+  protected WebDriverWait wait;
+
+  public AbstractPage(WebDriver driver) {
```

Comment:
Why is it `public`?

Reply:
fixed. it is more correct to do with protected constructor